_locales/_untranslated_en.json

```diff
@@ -22,5 +22,8 @@
   },
   "pref_show_notification_content_explain": {
     "message": "Shows sender and first words of message content in notification"
+  },
+  "ask_delete_message_desktop": {
+    "message": "Are you sure you want to delete message?"
```

Comment:
I think it should be "Are you sure you want to delete **the** message?" or "Are you sure you want to delete **this** message?"

Reply:
Fixed it. Thanks.